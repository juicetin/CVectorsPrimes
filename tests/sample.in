4 8
3 12 24
vector= primes 2
vector= random 1
vector= uniform 1
vector= sequence 1 1
vector= sequence 1 2
vector= reversed 0
vector= ascending 3
vector= descending 3
vector= scalar#add 2 2
vector= scalar#mul 2 2
vector= vector#add 2 2
vector= vector#mul 2 2
compute display 0
compute display 1
compute display 2
compute display 3
compute display 4
compute display 5
compute display 6
compute display 7
compute display 8
compute display 9
compute sum 0
compute sum 1
compute sum 2
compute sum 3
compute sum 4
compute sum 5
compute mode 3
compute median 3
compute minimum 3
compute maximum 3
compute element 0 0
compute element 0 1
compute frequency 2 0
compute frequency 2 1
