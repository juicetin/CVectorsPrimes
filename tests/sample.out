vector~0 :: 2 3 5
vector~1 :: 41 18467 6334
vector~2 :: 1 1 1
vector~3 :: 1 2 3
vector~4 :: 1 3 5
vector~5 :: 5 3 2
vector~6 :: 1 2 3
vector~7 :: 3 2 1
vector~8 :: 3 3 3
vector~9 :: 2 2 2
sum of vector~0 = 10
sum of vector~1 = 24842
sum of vector~2 = 3
sum of vector~3 = 6
sum of vector~4 = 9
sum of vector~5 = 10
mode of vector~3 = -1
median of vector~3 = 2
minimum of vector~3 = 1
maximum of vector~3 = 3
element 0 in vector~0 = 2
element 1 in vector~0 = 3
frequency of 0 in vector~2 = 0
frequency of 1 in vector~2 = 3
